fixed-hash/Cargo.toml

```diff
@@ -20,7 +20,7 @@ serde_derive = { version = "1.0", optional = true }
 libc = { version = "0.2", optional = true, default-features = false }
 
 [features]
-default = ["libc"]
+default = ["std", "libc"]
```

Comment:
The `serialize` feature should be removed too I think.

Reply:
Could you elaborate on that? As far as I see it may be useful when depending crates are using `serde`.

Reply:
True, and since you can simply annotate the hash types in `construct_hash!` macro ínvokation you really do not need serde support. You are absolutely right and I am going to remove it. :)